src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+
+function averageAge(people) {
+  return people
+    .map(person => person.died - person.born)
+    .reduce((sum, x) => sum + x)
+    / people.length;
+}
```

Comment:
you can avoid map here, try to do it just with reduce
And again, no `x` pls, it hurts me so bad 🥲